rllib/policy/torch_policy.py

```diff
@@ -842,6 +842,16 @@ def optimizer(
             optimizers = [
                 torch.optim.Adam(self.model.parameters(), lr=self.config["lr"])
             ]
+            if self.config["swa"]:
```

Comment:
This is awesome! Thanks Steven for the research and this PR here.
* Could we add an informative error here iff torchcontrib is not installed?

Reply:
Sure, patch incoming